plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/CameraAnimationsPluginImpl.kt

```diff
@@ -177,14 +177,18 @@ class CameraAnimationsPluginImpl : CameraAnimationsPlugin {
     handler.removeCallbacks(commitChangesRunnable)
   }
 
+  private fun CameraOptions.isEmpty() =
+    center == null && bearing == null && pitch == null && padding == null && zoom == null && anchor == null
```

Comment:
how not to forget update this method? maybe move to `CameraOptions.kt`?

Reply:
yep